### PR TITLE
Use symfony/translation for L10N plurals

### PR DESCRIPTION
**Pull request by rullzer**
Let use a library to properly do this 

Requires: 
 - [x] https://github.com/nextcloud/3rdparty/pull/61

### CODE REVIEW

File: lib/private/L10N/L10N.php [hunk 1]
```diff
@@ -191,8 +192,12 @@ public function getTranslations() {
 	 */
 	public function getPluralFormFunction() {
 		if (is_null($this->pluralFormFunction)) {
-			$this->pluralFormFunction = $this->factory->createPluralFunction($this->pluralFormString);
+			$lang = $this->getLanguageCode();
```
Comment (nickvergessen) [inline lib/private/L10N/L10N.php]: I guess we need to translate our de_DE cases?

**Review by nickvergessen — DISMISSED** [review]

File: lib/private/L10N/L10N.php (hunk 1 above)
Comment (rullzer) [inline lib/private/L10N/L10N.php]: Nope: https://github.com/nextcloud/3rdparty/compare/107e29e4c81d6b255abee5bc77fef3040df6ca07...d39f87b83172e5201bda222d4f33999df3378566#diff-fe1183349c109e03c80b74fd4b609436R39

**Comment by rullzer** [comment]
3rdparty merged. Updated!

**Comment by MorrisJobke** [comment]
Any  reason not to also remove the createPluralFunction method in the factory?

**Review by MorrisJobke — APPROVED** [review]
Works 👍

**Review by nickvergessen — APPROVED** [review]
Works, tested a language with 1, 2 and 6 rules

**Comment by nickvergessen** [comment]
@MorrisJobke MySQL has gone away, but I guess we can merge?

**Comment by MorrisJobke** [comment]
> @MorrisJobke MySQL has gone away, but I guess we can merge?

Yep - because it ran into the 1:40 h timeout :/

**Pull request merged**